docs/security/08-02-update-tls-certificates.md

```diff
@@ -22,19 +22,46 @@ The TLS certificate is a vital security element. Follow this tutorial to update
 
   >**CAUTION:** When you regenerate the TLS certificate for Kyma, the `kubeconfig` file generated through the Console UI becomes invalid. To complete these steps, use the admin `kubeconfig` file generated for the Kubernetes cluster that hosts the Kyma instance you're working on.
 
-  1. Edit the `owndomain-overrides` ConfigMap and replace the existing certificate and key found in the **global.tlsCrt** and **global.tlsKey** with a new, base64-encoded pair.
+  1. Delete the `net-global-overrides` ConfigMap.
 
+    ```bash
+    kubectl delete cm -n kyma-installer net-global-overrides
     ```
-    kubectl edit cm -n kyma-installer owndomain-overrides
+
+  2. Export the Kyma version, your domain, new certificate and key as the environment variables.
+
+    ```bash
+    export KYMA_VERSION={KYMA_RELEASE_VERSION}
+    export DOMAIN={YOUR_DOMAIN}
+    export TLS_CERT={YOUR_NEW_CERTIFICATE}
+    export TLS_KEY={YOUR_NEW_KEY}
+    ```
+
+  3. Trigger the update process. Run:
+
+    ```bash
+    kyma upgrade -s $KYMA_VERSION --domain $DOMAIN --tls-cert $TLS_CERT --tls-key $TLS_KEY
     ```
 
-    Delete the Secret that stores the expired certificate:
+    To watch the progress of the update, run:
```

Comment:
You don't need to use an additional command to watch the progress of the update. `kyma` CLI shows you the progress and terminates when Kyma is installed.

Reply:
Removed :)